android/delegate.gradle

```diff
@@ -0,0 +1,153 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.3.+'
+        classpath 'com.jakewharton.sdkmanager:gradle-plugin:0.12.+'
+        classpath 'com.github.dcendents:android-maven-plugin:1.2'
+    }
+}
+apply plugin: 'android-sdk-manager'
+apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'signing'
+
+def properties = new Properties()
+properties.load(new FileInputStream(file('../version.properties')))
+
+description = 'gRPC: Android'
+group = "io.grpc"
+version = "${properties.version}"
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+project.archivesBaseName = "grpc-android"
```

Comment:
Unnecessary? since `project.archivesBaseName which in turn defaults to project.name`.